app/src/test/java/org/simple/clinic/summary/PatientSummaryUpdateTest.kt

```diff
@@ -71,8 +71,10 @@ class PatientSummaryUpdateTest {
 
   @Test
   fun `when there are patient summary changes and at least one measurement is present, clicking on back must show the schedule appointment sheet`() {
+    val model = defaultModel.copy(currentFacility = facility)
```

Comment:
`PatientSummaryModel` already has a method, `currentFacilityLoaded()`. Could you use that instead since that is the canonical way to update the model with the facility?